### PR TITLE
Publish version 0.0.7

### LINKED ISSUE #181: Publish 0.0.7 of the SDK (and child packages) for use in the Decentraland demo app

### PR DESCRIPTION
**Pull request by marcelomorgado**
### Issue link

#181 

### Auto-close the issue?

Closes #181 

### Types of changes

Bug fix (non-breaking change that fixes an issue)

New feature (non-breaking change that adds functionality)

Technical debt (a code change that doesn't fix a bug or add a feature but makes something clearer for devs)

### CODE REVIEW

**Comment by marcelomorgado** [comment]
Note: I've created a publish script.

File: package.json [hunk 1]
```diff
@@ -10,6 +10,7 @@
     "prettier-plugin-solidity": "^1.0.0-alpha.12"
   },
   "scripts": {
+    "publish": "npx lerna run clean:3rd-parties --scope tasit-contracts && npx lerna publish",
```
Comment (pcowgill) [inline package.json]: `npx lerna publish` worked for you directly? I know before we had been using `npx lerna public from-package`

What prompts did you get when going this route?

File: packages/tasit-contracts/decentraland/scripts/clean.sh [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+fi
+
+
+DECENTRALAND_DIR="$PROJECT_DIR/decentraland";
+REPOS="mana land marketplace-contracts"
+
+for repo in $REPOS;
+do
+    REPO_DIR="$DECENTRALAND_DIR/$repo";
+    cd $REPO_DIR && git reset --hard && git clean -f
```
Comment (pcowgill) [inline packages/tasit-contracts/decentraland/scripts/clean.sh]: Nice, this script is very useful

File: packages/tasit-action/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "tasit-action",
-  "version": "0.0.6",
+  "version": "0.0.7",
```
Comment (pcowgill) [inline packages/tasit-action/package.json]: Did you end up having to manually increment these, or did lerna do it for you?

**Comment by pcowgill** [comment]
Yeah, the publish script looks great. Thanks.

Looks like it worked with the new permissions! https://www.npmjs.com/package/tasit-sdk

**Comment by pcowgill** [comment]
@marcelomorgado Merging this, but I'll still be curious about the open threads after it's merged. Thanks!

File: package.json (hunk 1 above)
Comment (marcelomorgado) [reply package.json]: That works. After the run, shows possible versions, I've selected patch 0.0.7 and then as final prompt a list of packages to upgrade.

File: packages/tasit-action/package.json (hunk 3 above)
Comment (marcelomorgado) [reply packages/tasit-action/package.json]: Lerna did

**Pull request merged**